packages/reva-app/src/services/candidacyServices.ts

```diff
@@ -328,7 +328,7 @@ const CANDIDACY_SELECTION = `
         status
         isActive
       }
-      regionId
+      departmentId
```

Comment:
Vu que j'expose le department finalement, on va prendre l'id dans l'object `department`

```suggestion
      department {
        id
      }
```